os/arch/arm/src/armv8-m/up_hardfault.c

```diff
@@ -83,7 +87,11 @@
  */
 
 #ifdef CONFIG_DEBUG_HARDFAULT
+#ifdef CONFIG_SECURITY_LEVEL
+#define hfdbg(format, ...) if (get_security_level() == LOW_SECURITY_LEVEL) lldbg(format, ##__VA_ARGS__)
```

Comment:
It would be good if we can implement a new debug macro like hfdbg for all fault debug statements. In fact, we can rename hfdbg as flt_dbg and use it in all fault handling files.

Reply:
Thank you for review~
I heard that we should always keep the assert logs print on.

So I removed hfdbg and chaged lldbg(assertdbg).